psi4/src/export_mints.cc

```diff
@@ -1128,6 +1128,7 @@ void export_mints(py::module& m) {
                                "The character symbol for the angular momentum of the given contraction")
         .def_property_readonly("AMCHAR", py::cpp_function(&GaussianShell::AMCHAR),
                                "The upper-case character symbol for the angular momentum of the given contraction")
+        .def("coord", &GaussianShell::coord, "Returns ith coordinate this shell is on.") 
```

Comment:
can this be readonly like everything else in the class export?

Reply:
It probably should be, yes. I'll fix that up!